trailblazer/services/analysis_service/utils.py

```diff
@@ -5,31 +5,23 @@
 from trailblazer.store.models import Analysis
 
 
-def get_status_count(analyses: list[Analysis], status: TrailblazerStatus) -> int:
-    return len([a for a in analyses if a.status == status])
-
-
-def get_delivered_count(analyses: list[Analysis]) -> int:
-    return len([analysis for analysis in analyses if analysis.delivery])
+def get_status_counts(analyses: list[Analysis]) -> dict[TrailblazerStatus, int]:
+    """Returns the amount of analyses with each status."""
+    delivered: int = 0
+    status_counts: dict = {status: 0 for status in TrailblazerStatus}
```

Comment:
Should delivered be part of TrailblazerStatus? Or are these two separate worlds?

Reply:
That is the million dollar question. Since there was a need for tracking when and by whom it was delivered, it did not fit into the status column. There was also wishes that the TrailblazerStatus only referred to the analysis status, not to the handling after the analysis has been completed. So for now, no, but I am not opposed to a change at some point.

Reply:
Sidetrack, `TrailblazerStatus` is a really non descriptive name that creates confusion 😅 It is unclear what it is meant to represent.

Reply:
Out of scope for this PR though 😅